core/ts/MaciState.ts

```diff
@@ -1334,6 +1330,82 @@ class Poll {
         }
         return true
     }
+    
+    /**
+     * Serialize the Poll object to a JSON object
+     * @returns a JSON object
+     */
+    toJSON() {
+        return {
+            duration: this.duration,
+            pollEndTimestamp: this.pollEndTimestamp.toString(),
+            coordinatorKeypair: this.coordinatorKeypair.toJSON(),
+            treeDepths: this.treeDepths,
```

Comment:
Hmm.. I'd prefer not to write the coordinator keypair to the file.

Reply:
Fair point, I think we can skip saving that on a file and have it passed as argument to the 'fromJSON' method